core/trino-main/src/main/java/io/trino/execution/QueryTracker.java

```diff
@@ -55,7 +54,7 @@ public class QueryTracker<T extends TrackedQuery>
     private final int maxQueryHistory;
     private final Duration minQueryExpireAge;
 
-    private final ConcurrentMap<QueryId, T> queries = new ConcurrentHashMap<>();
+    private final ConcurrentMap<QueryId, T> queries = new ConcurrentHashMap<>(0);
```

Comment:
Seems unrelated change.
Also, why bother ? I don't think we will have many instances of this class and having more than 16 queries tracked is not unusual